server/data/models/__mocks__/roleModel.js

```diff
@@ -0,0 +1,7 @@
+const create = require('../../../tests/util/create');
+
+const Role = {
```

Comment:
I'm guessing we don't need this model, for the same reason, that it won't be exposed? Or do we somehow need it to get the foreign key for the users to roles table?